src/components/UserInfo.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { User } from '../types/User';
+
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = (props) => {
+  const { user } = props;
+  const { name, email } = user;
+
+  return (
+    <>
```

Comment:
```suggestion
    <>
```
Better to use something more semantic